decimal-macros/examples/test.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(plugin)]
+#![plugin(decimal_macros)]
+extern crate decimal;
+
+fn main() {
+    let a = d128!(0.1);
```

Comment:
How about better tests?

d128!(0.1) should be equal to 1 as dec128 / 10 as dec128, etc